packages/web/src/pages/sign-in-page/MetamaskSignInButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react'
+
+import { Button, ButtonType, IconMetamask } from '@audius/harmony'
+
+import { useNavigateToPage } from 'hooks/useNavigateToPage'
+import { userHasMetaMask } from 'pages/sign-up-page/utils/metamask'
+import { FEED_PAGE } from 'utils/route'
+
+const messages = {
+  signIn: 'Sign In With MetaMask'
+}
+
+export const MetamaskSignInButton = () => {
+  const navigate = useNavigateToPage()
+
+  const handleClick = useCallback(() => {
+    try {
+      window.localStorage.setItem('useMetaMask', JSON.stringify(true))
+    } catch (err) {
+      console.error(err)
```

Comment:
any risk of leaking stack traces here? I guess that's less of a concern when you're open source 😅

Reply:
This was copied from existing code but agreed it's a bit strange